css/style.css

```diff
@@ -0,0 +1,51 @@
+.header {
+  height: 2.5em;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.btn-icon {
+  background: transparent;
+  border: none;
+}
+
+.home {
```

Comment:
- [ ] Please you did not fix what the previous review noticed. 
- See your design below: 
- 
![ok](https://user-images.githubusercontent.com/63972670/164555267-e183da65-6fc0-43a2-a6a1-623aa836dff5.PNG)

- [ ] I suggest that you use "padding 3rem 2rem" or something similar to fix the home section. " padding-left: 2rem" for the header section or something else which can fix it.
- [ ] please push the "hamburger" button to the right side. "margin or padding" can also fix it.
- [ ] Here is the link to the [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).